tests/unit/views/notifications/test_project_crew_notification.py

```diff
@@ -0,0 +1,200 @@
+"""Test template strings in project crew membership notifications."""
+# pylint: disable=too-many-arguments
+
+from pytest_bdd import given, parsers, scenarios, then, when
+
+from funnel import models
+from funnel.models.membership_mixin import MEMBERSHIP_RECORD_TYPE
+
+scenarios("project_crew_notification.feature")
+
+
+@given(
+    "Rincewind and Twoflower are project crew in the project Expo 2010",
+    target_fixture='rincewind_editor',
+)
+def add_and_check_project_crew_members(
+    db_session,
+    client,
+    login,
+    user_vetinari,
+    user_rincewind,
+    user_twoflower,
+    project_expo2010,
+) -> models.ProjectCrewMembership:
+    rincewind_editor = models.ProjectCrewMembership(
+        parent=project_expo2010,
+        user=user_rincewind,
+        is_editor=True,
+        granted_by=user_vetinari,
+    )
+    db_session.add(rincewind_editor)
+    db_session.commit()
+    assert 'editor' in project_expo2010.roles_for(user_rincewind)
+    assert 'editor' in project_expo2010.roles_for(user_vetinari)
+    return rincewind_editor
+
+
+@when(
+    parsers.parse("Vetinari adds twoflower with role {role}"),
```

Comment:
For this fixture to be re-usable (which you want), the `when` statement must also include the project name.